src/locator.ts

```diff
@@ -8,15 +9,35 @@ import {
 	window,
 	workspace,
 } from "vscode";
+import { Utils } from "vscode-uri";
 import type Biome from "./biome";
 import {
 	platformIdentifier,
 	platformSpecificBinaryName,
 	platformSpecificNodePackageName,
 } from "./constants";
-import { config, fileExists, getLspBin } from "./utils";
+import { config, fileExists, getLspBin, safeSpawnSync } from "./utils";
 
 export default class Locator {
+	private get globalNodeModulesPaths(): Record<string, Uri | undefined> {
+		const npmGlobalNodeModulesPath = safeSpawnSync("npm", ["root", "-g"]);
+		const pnpmGlobalNodeModulesPath = safeSpawnSync("pnpm", ["root", "-g"]);
+		const bunGlobalNodeModulesPath = Utils.resolvePath(
+			Uri.file(homedir()),
+			".bun/install/global/node_modules",
```

Comment:
This assumes that the user has one of these tools installed. In case none of them are installed, would they throw an error that we need to catch?

Reply:
`safeSpawnSync` will handle the failure gracefully. If the command does not exist or exits with an error, the path will be undefined, which is handled.